setup.py

```diff
@@ -34,7 +34,7 @@
     description="This package allows to list Mbed OS devices which are connected to the host computer.",
     keywords="Arm Mbed OS MbedOS devices list",
     include_package_data=True,
-    install_requires=["python-dotenv", "Click==7.0", "mbed-targets"],
+    install_requires=["python-dotenv", "Click==7.0", "mbed-targets", "mbed-tools-lib"],
```

Comment:
```suggestion
    install_requires=["python-dotenv", "Click==7.0", "mbed-targets"],
```
Could you remove this for now as it is not a dependency yet?